### ISSUE TITLE
Disable the dragging corner

### ISSUE THREAD

**Issue by librarian-ape** [jspreadsheet/ce#996]
Hi Paul,

thank for your work so far. I was just wondering if it would be possible to add an option to disable the dragging corner to prevent users to copy data across adjacent cells of different types.

regards.

**Comment by pphod** [comment]
You can start your table with -> selectionCopy: false to disable the corner. But, this would be a total disable. There is no intermediate option at this point.

When you have time, can you take a look on Jexcel Pro. We can offer custom support ;)
https://jexcel.net/v5

**Comment by librarian-ape** [comment]
hi paul, thanks for your answer. I'll have a deeper look on the Pro version. In the meanwhile, I've tried the option you suggest, but it doesn't seem to work the proper way (I'm on angular 6 and have installed Jexcel with npm). Even if I set selectionCopy to false, the corner will still be available and allow copy by dragging.
I've dug into the code and made a few modifications to take this parameter in account, disabling totally obj.corner (letting it set to null and testing against it's value everytime it is used) if selectionCopy is set to false.
I've attached my local version of jexcel.js (as a txt file =>[jexcel.txt](https://github.com/paulhodel/jexcel/files/4728633/jexcel.txt)) to this message.

thanks again for this.
